start.php

```diff
@@ -83,11 +83,25 @@ function_exists( 'wp_is_json_request' ) &&
      */
 	$themes_directory         = get_theme_root( get_stylesheet() );
 	$themes_directory_name    = basename( $themes_directory );
-	$theme_candidate_basename = basename( dirname( $fs_root_path ) ) . '/' . basename( $fs_root_path );
 
-	if ( $file_path == fs_normalize_path( realpath( trailingslashit( $themes_directory ) . $theme_candidate_basename . '/' . basename( $file_path ) ) )
-	) {
-		$this_sdk_relative_path = '../' . $themes_directory_name . '/' . $theme_candidate_basename;
+	// This change ensures that the condition works even if the SDK is located in a subdirectory (e.g., vendor)
+	$theme_candidate_basename = str_replace($themes_directory . '/' . get_stylesheet() . '/', '', $fs_root_path );
```

Comment:
@DanieleAlessandra While I understand your logic, I think the placement of it can be made better.

The variable `$theme_candidate_basename` means the "basename" of the theme itself, it doesn't need to be the basename of the SDK directory inside the theme.

So either

1. Rename this variable to something like `$theme_candidate_sdk_basename`
2. Or change the logic of `$is_current_active_theme`